.github/workflows/ci.yml

```diff
@@ -135,8 +135,19 @@ jobs:
           version: "0.6.12"
           enable-cache: true
 
+      - name: Install dependencies
+        run: uv sync
+
+      - name: Install typing-extensions
+        if: typing-extensions != 'latest'
+        run: uv pip install typing-extensions==${{ matrix.typing-extensions }}"
+
+      - name: Install typing-extensions
+        if: typing-extensions == 'latest'
+        run: uv pip install typing-extensions --upgrade --prerelease=allow
+
       - name: Test
-        run: uv run --with="typing-extensions==${{ matrix.typing-extensions }}" -- python -m pytest tests/unit/test_typing.py
+        run: uv run --no-sync -- python -m pytest tests/unit/test_typing.py
```

Comment:
```suggestion
        run: |
          if [[ "${{ matrix.typing-extensions }}" == 'latest' ]]; then
            uv run --with=typing-extensions -- python -m pytest tests/unit/test_typing.py
          else
             uv run --with="typing-extensions==${{ matrix.typing-extensions }}" -- python -m pytest tests/unit/test_typing.py
          fi
```

Reply:
Mh. Not sure if I find this more readable?

Reply:
Both are fine :)

Reply:
It can even be simplier with `bash`

Reply:
That won't work tho as ` uv run --with="typing-extensions=="` isn't valid. You need the different cmds in this case; Unfortunately there's no way (that I know of) to use `--with` to install and upgrade to the latest version

Reply:
hm, I solved `==` issue, but I am not sure about the "update" part :)
anyways, let's not spend more time on this. if your version works - let's use it :_